roles/k8s_best_practices_certsuite/tasks/tests.yml

```diff
@@ -11,6 +11,16 @@
     state: directory
     mode: '0755'
 
+- name: Verify if the CWE Hydra API key file exists
+  stat:
+    path: "{{ kbpc_cwe_apikey_path }}"
+  register: kbpc_cwe_apikey_file
+
+- name: Set kbpc_cwe_apikey if the token file exists
+  set_fact:
+    kbpc_cwe_apikey: "{{ lookup('file', kbpc_cwe_apikey_path) }}"
+  when: kbpc_cwe_apikey_file.stat.exists
+
```

Comment:
It would be great if the API offered a way to validate the provided key is valid

Reply:
they do not :/ That's why did not want Hydra being used :(